src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -38,11 +38,24 @@ public function prepend(ContainerBuilder $container)
             return;
         }
 
-        if (!isset($frameworkConfiguration['serializer'], $frameworkConfiguration['serializer']['enabled'])) {
+        $serializerConfig = null;
+        $propertyInfoConfig = null;
+
+        foreach ($frameworkConfiguration as $frameworkParameters) {
+            if (isset($frameworkParameters['serializer'])) {
+                $serializerConfig = $serializerConfig ?: $frameworkParameters['serializer'];
+            }
+
+            if (isset($frameworkParameters['property_info'])) {
+                $propertyInfoConfig = $propertyInfoConfig ?: $frameworkParameters['property_info'];
+            }
+        }
+
+        if (null === $serializerConfig || !isset($serializerConfig['enabled'])) {
```

Comment:
The `null` test is useless, isset will already take care of that. It can be removed.